### ISSUE TITLE
Talkback.php - Add config vars for google recaptcha and slack 

### ISSUE THREAD

**Issue by cgb37** [subjectsplus/SubjectsPlus#1152]
For the distributed version we need to add config vars for google recaptcha and slack webhooks

**Comment by cgb37** [comment]
Fixed [0dc6d3d]
